Bottom: f208998c0e8da6055853819e5704f90e6fd45c3c
Top:    7cd1ae6d85925477bab8275bf75a26d085a3d8ec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 17:58:33 +0000

hid/lpr: Don't mis-match popen and fclose (should be pclose)

(Caught with cppcheck)


---

diff --git a/src/hid/lpr/lpr.c b/src/hid/lpr/lpr.c
index 259812c..fc9403b 100644
--- a/src/hid/lpr/lpr.c
+++ b/src/hid/lpr/lpr.c
@@ -106,7 +106,7 @@ lpr_do_export (HID_Attr_Val * options)
 
   ps_hid_export_to_file (f, options);
 
-  fclose (f);
+  pclose (f);
 }
 
 static void
